src/category_theory/monoidal/internal.lean

```diff
@@ -99,11 +99,26 @@ instance : category (Mon_ C) :=
 @[simp] lemma comp_hom' {M N K : Mon_ C} (f : M ⟶ N) (g : N ⟶ K) :
   (f ≫ g : hom M K).hom = f.hom ≫ g.hom := rfl
 
+variables (C)
+
 /-- The forgetful functor from monoid objects to the ambient category. -/
 def forget : Mon_ C ⥤ C :=
```

Comment:
The forgetful functor here takes `C` as an explicit argument - I think this is a good idea but iirc there are a couple of times where the forgetful functor has no explicit arguments, do we want to be consistent about this throughout?

Reply:
Yes, that would be good. I'm pretty sure I want it explicit here.

I did this separately as #4139.